### PR TITLE
CTD-1855 fix UserAlias error

### PR DESCRIPTION
**Pull request by DashaSyvyuk**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,6 +14,6 @@ setenv =
 passenv = COVERALLS_REPO_TOKEN
 
 commands =
-    flake8
+    flake8 --exclude node_modules,venv,.serverless
```
Comment (tomrutherford-livestyled) [inline tox.ini:17]: please remove this change

File: livestyled/tests/fixtures/mock_responses/ls_api/user_1234.json [hunk 2]
```diff
@@ -50,5 +50,14 @@
...
+    "userAliases": [
+        {
+            "@id": "/user_management/user_aliases/510",
+            "@type": "UserAlias",
+            "id": 510,
+            "userAliasType": "/user_management/user_alias_types/4",
+            "value": "test1234"
+        }
     ]
 }
```
Comment (tomrutherford-livestyled) [inline livestyled/tests/fixtures/mock_responses/ls_api/user_1234.json:63]: add new line

File: livestyled/schemas/tests/fixtures/example_user.json [hunk 3]
```diff
@@ -68,5 +68,14 @@
...
+    "userAliases": [
+        {
+            "@id": "/user_management/user_aliases/510",
+            "@type": "UserAlias",
+            "id": 510,
+            "userAliasType": "/user_management/user_alias_types/4",
+            "value": "test1234"
+        }
+    ]
 }
```
Comment (tomrutherford-livestyled) [inline livestyled/schemas/tests/fixtures/example_user.json:81]: add new line

File: livestyled/models/user.py [hunk 4]
```diff
@@ -292,14 +292,16 @@ def placeholder(
...
-            updated_at: datetime,
-            created_at: datetime
+            user: str or None = None,
+            updated_at: datetime or None = None,
+            created_at: datetime or None = None
     ):
+        self.id = id
         self.user_alias_type = User.placeholder(int(user_alias_type))
-        self.user = User.placeholder(int(user))
+        self.user = User.placeholder(int(user)) if user else None
```
Comment (tomrutherford-livestyled) [inline livestyled/models/user.py:304]: How can the user alias exist without a user?

**Review by tomrutherford-livestyled — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (DashaSyvyuk) [reply tox.ini]: done

File: livestyled/models/user.py (hunk 4 above)
Comment (DashaSyvyuk) [reply livestyled/models/user.py:304]: if we make request `/v4/user_manager/users/{id}`
we get in request ```"userAliases": [
                {
                    "@id": "/user_management/user_aliases/510",
                    "@type": "UserAlias",
                    "id": 510,
                    "userAliasType": "/user_management/user_alias_types/4",
                    "value": "test1234"
                }
            ]```

File: livestyled/tests/fixtures/mock_responses/ls_api/user_1234.json (hunk 2 above)
Comment (DashaSyvyuk) [reply livestyled/tests/fixtures/mock_responses/ls_api/user_1234.json:63]: done

File: livestyled/models/user.py (hunk 4 above)
Comment (tomrutherford-livestyled) [reply livestyled/models/user.py:304]: so why do we even need the user property at all?

**Review by tomrutherford-livestyled — CHANGES_REQUESTED** [review]

File: livestyled/models/user.py (hunk 4 above)
Comment (DashaSyvyuk) [reply livestyled/models/user.py]: removed, as we don't have any `get_user_alias` method for now

**Review by tomrutherford-livestyled — APPROVED** [review]

**Pull request merged**